### DISCUSSION TITLE
Blood composition

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1254] [Questions & Problems]
I am trying to model a compound that interacts strongly with proteins. I would like to introduce into the model individually measured values of parameters such as:  Total blood protein, blood albumin, hemoglobin, RBC in addition to hematocrit, which quite clearly can be introduced in creating an individual.
I would be grateful for assistance in introducing such parameters into an individual's building block

**Comment by StephanSchaller** [comment]
Hi @zt1947, as a quick rough solution, you could use the protein scaling factor (to adjust for changes in plasma protein (also used for the ontogeny) and the hematocrit.
Otherwise, you would have to move to MoBi and create these parameters, but more importantly, you would have to adapt the ADME processes to integrate these parameters to have an effect... depending on what you are trying to do, this could be quite complex.

Best, Stephan

**Comment by zt1947** [comment]
Thanks!
The situation I am trying to model, is a real life one, based upon
monitoring patients undergoing quite drastic pharmacological procedures.
Therefore each of the actually observed levels (albumin, RBC, hemoglobin
and other blood components) may differ from the standard values included in
the QSP database by a different scaling factor. How do I get around this?
The drug I am trying to model may bind irreversibly to blood proteins. I
have good indications that exposure to the iv-administered drug correlates
with albumin levels. I will initially try to include an irreversible
binding to albumin in a MoBi model. Does this make any sense and how
complex may this be.
Sincerely
Zvi
